control_plane/roles/control_plane_device/tasks/mngmnt_network_container_image.yml

```diff
@@ -29,8 +29,8 @@
 - name: Update omnia project path in k8s_mngmnt_network.yml
   replace:
     path: "{{ role_path }}/files/k8s_mngmnt_network.yml"
-    regexp: 'path: /root/omnia'
-    replace: "path: {{ role_path.split('control_plane')[0] }}"
+    regexp: '        - name: omnia-storage\n          hostPath:\n            path:.*'
```

Comment:
@blesson-james, if this play is always executed, why not simply alter `omnia/control_plane/roles/control_plane_device/files/k8s_mngment_network.yml` instead of using this regular expression?

Reply:
@lwilson this task is for actually finding the directory path where user has cloned omnia (which is not fixed as user can clone it any directory and give it any name) and replacing it with the volume path present in k8s yaml file, since we need to mount the cloned omnia directory as a volume inside k8s pods.

Reply:
sounds good, @blesson-james.